.github/workflows/test.yml

```diff
@@ -16,14 +16,16 @@ jobs:
   linux:
     name: Testing ${{ matrix.os }} GHC-${{ matrix.ghc }}
     runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ matrix.allow-failure }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-22.04, macos-14]
+        os: [ubuntu-22.04]
         ghc: [9.4.8, 9.6.6, 9.8.2]
         cabal: [3.10.3.0]
-        allow-failure: [false]
-      fail-fast: false
+        include:
+          - os: macos-14
+            ghc: 9.8.2
+            cabal: 3.10.3.0
```

Comment:
I think the intent here is to include a single macOS configuration that tests the latest supported GHC version. Is that right? If so, it would be helpful to clarify this intent in a comment.